src/assets/styles/custom-scroll.less

```diff
@@ -4,6 +4,8 @@
 
 .side-scroll {
   padding-bottom: 0px;
+  border-top: 1px solid @dark-background;
+  transition: border 0.7s;
```

Comment:
@mic-smith Doesn't that selector create the top border that we do have on the search tab?